inspirehep/modules/workflows/static/js/inspire_workflows_ui/holdingpen/holdingpen.filters.js

```diff
@@ -85,10 +85,27 @@
     }
   }
 
+  function unifyKeywordsFilter() {
+    return function (classifier_results) {
+      var filteredCoreKeywords = classifier_results['filtered_core_keywords'] || [];
+      var coreKeywords = classifier_results['core_keywords'] || [];
+      var keywords = filteredCoreKeywords.concat(coreKeywords)
+        .map(function(keyword) {
+          return keyword['keyword']
+        });
+
+      return Array.from(new Set(keywords));
```

Comment:
I guess this to remove duplicates, correct? How these keywords are going to be sorted then?